src/service/provider/provider.service.ts

```diff
@@ -59,8 +70,8 @@ export class ProviderService {
         },
       ]
     >`select count(distinct provider)::int,
-             100 * avg(success_rate) as "averageSuccessRate"
-      from provider_retrievability_daily;`;
+             100 * avg(avg_retrievability_success_rate) as "averageSuccessRate"
+      from providers_weekly where week = ${lastWeek.week};`;
```

Comment:
We should be taking only last full week. So if we're on Tuesday Oct 1, we should take week Sep 23 - Sep 29.
This query would take week Sep 30 -  Oct 6, which isn't full and isn't a good week to take average from.